Cauli/Configuration/RecordSelector.swift

```diff
@@ -35,7 +35,7 @@ public struct RecordSelector {
     }
 }
 
-extension RecordSelector {
+public extension RecordSelector {
```

Comment:
Well, now the RecordSelector is public, but the `max` and `all` functions are still internal.

Reply:
Actually, the functions have the same visibility as the class unless differently defined. In this case they are public. Would you rather like to see them explicitly public?

Reply:
🤦‍♂️ your right

Reply:
Approved? Or do you want me to change it?